### ISSUE TITLE
/docs repaints multiple times during page load

### ISSUE THREAD

**Issue by ghuntley** [gitpod-io/website#643]
https://user-images.githubusercontent.com/127353/122509837-cfa6c280-d047-11eb-827e-853fa5b6e5d5.mp4

**Comment by mikenikles** [comment]
That's due to the fact that we made a compromise when we developed the website and mixed traditional CSS in external files with CSS inlined into the Svelte components.

The proper fix for this is to remove all external custom CSS and use Tailwind CSS classes like we do with the Gitpod product.

**Comment by nisarhassan12** [comment]
Relates to #648

**Comment by ChristinFrohne** [comment]
What's the status here @nisarhassan12 ? Please move it to the [DCS inbox](https://github.com/orgs/gitpod-io/projects/4) if that's something you can fix

**Comment by nisarhassan12** [comment]
It is already a work in progress  It relates to https://github.com/gitpod-io/website/issues/648 which I'm making progress in bit by bit every week in other Prs. 

@ghuntley Can you test and tell if it has gotten any better with the progress that we have made on #648 ? Thanks

**Comment by nisarhassan12** [comment]
The video that you shared here https://github.com/gitpod-io/website/issues/643#issue-924549830 looks like a video recorded from a Gitpod workspace in development mode where their entire Tailwind CSS library is shipped without any tree shaking with a size more than 5.5 MB which is insanely huge. 

I'm unable to reproduce this in prod and there the size of the `global.css` is `5.3KB`. 

I'm unable to reproduce this I tried to slow my internet connection via Network throttling(I set it to Regular 3G).
